plugin/bufferhint.vim

```diff
@@ -97,7 +101,7 @@ fu! bufferhint#Popup()
         sy clear
         sy match KeyHint /^../
         sy match AtHint /@/
-        hi clear
+        "hi clear
```

Comment:
LineNr highlight seems correct after changing to 'hi clear KeyHint', need I make another pull request with it ?